core/app/assets/config/refinery_core_manifest.js

```diff
@@ -5,10 +5,11 @@
 //= link refinery/nestedsortables.js
 //= link refinery/serializelist.js
 //= link refinery/tree.js
+//= link refinery/wymeditor.js
```

Comment:
You should not add the WYMeditor asset here because Refinery is decoupled from its WYSIWYG. I've updated the feature/rails-5 branch of refinerycms-wymeditor with the wanted fix: https://github.com/parndt/refinerycms-wymeditor/pull/40/commits/17748af46dfca24603d3e4683be66230ab1431fd